content/post/116-protoschool-merkle-dags.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2021-01-10
+url: /2021-01-14-protoschool-tutorial-merkle-dags/
+title: Explore Merkle DAGs at ProtoSchool
+author: Teri Chadbourne
+tags: ipfs, ipld, protoschool
+header_image: 116-protoschool-merkle-dags.jpg
+---
+
+The [ProtoSchool](https://proto.school) team is pleased to announce the launch of a shiny new multiple-choice tutorial on [Merkle DAGs](https://proto.school/merkle-dags), the cornerstone of our freshly launched [IPLD course](https;//proto.school/course/ipld). 🎉
+
+This new content sprang from feedback on our Decentralized Data Structures tutorial, which originally explored both content addressing and data structures. Based on learner feedback, we've split the content in two! You'll now find the majority of our original content in our rebranded tutorial, [Content Addressing on the Decentralized Web](https://proto.school/content-addressing), which highlights the key differences in how data is shared and retrieved on the centralized web and decentralized web. Meanwhile, our brand new tutorial, [Merkle DAGs: Structuring Data for the Distributed Web](https://proto.school/merkle-dags), is chock full of new content exploring the intricacies the magical data structure that much of the decentralized web relies on.
+
+<img src="/116-protoschool-merkle-dags/dag-deduplication.png" alt="Merkle DAG diagram"/>
+
+_Screenshot of a Merkle DAG from the new tutorial_
+
+In IPFS, as across the decentralized web, content addressing enables us to securely locate and identify data stored by peers. The Content Identifiers (CIDs) we often use to link to individual files or pieces of data can also be used to express complex webs of data in their entirety, when we structure that data as content-addressable Merkle DAGs. Merkle DAGs, wrapped in structures provided by IPLD, are key to the interoperability of IPFS, libp2p, Filecoin, and other distributed protocols. In this new code-free ProtoSchool tutorial, we take a deep dive into the properties that make Merkle DAGs the backbone of the distributed web, and explore the many benefits that the data structure unlocks, from verifiability to distributability to deduplication. [Check it out!](https://proto.school/merkle-dags)
```

Comment:
"Merkle DAGs, wrapped in structures..." doesn't quite sound right to my ear. IPLD defines a structure for a Merkle DAG, more than than wrapping it.

Perhaps something closer to "Merkle DAGs, structured using schemas provided by IPLD..."?

Reply:
Thanks for the suggestion! Wasn't sure how to describe that relationship.